### DISCUSSION TITLE
Removing Stopwords from Confessions

### DISCUSSION THREAD

**Discussion by iharshit009** [iharshit009/SecretPeek#8] [Ideas]
Anonymity may give rise to Profanity, therefore we may need to either remove/replace them or don't allow them to post.
A list of Stopwords could be discussed. Suggestions in this regard are welcomed.

**Comment by Rits1272** [comment]
Should we use a Machine Learning Classifier to remove stopwords? @iharshit009

**Comment by iharshit009** [comment]
Please elaborate.

Harshit Jain
